hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/CodeCache.java

```diff
@@ -40,63 +31,22 @@
  * a value is stale is present.
  */
 @Singleton
-public class CodeCache {
-    private final EntityAccess entityAccess;
-    private final Cache<BytesKey, Code> cache;
-
+public class CodeCache extends AbstractCodeCache {
     @Inject
     public CodeCache(final NodeLocalProperties properties, final EntityAccess entityAccess) {
```

Comment:
You can substitute `EntityAccess` with `HederaEvmEntityAccess`, so that we don't limit the scope of this field.

Reply:
It seems we don't have much need of `CodeCache` since the functionalities are covered by `AbstractCodeCache`. You can rename `AbstractCodeCache` -> `CodeCache` and delete `CodeCache` from `hedera-mono-service`.